enterprise/security/src/main/java/org/neo4j/server/security/enterprise/configuration/SecuritySettings.java

```diff
@@ -129,7 +131,7 @@
             "The LDAP referral behavior when creating a connection. This is one of `follow`, `ignore` or `throw`.\n" +
```

Comment:
Bulleted lists don't work in this context. The output will be a paragraph with some asterisks in it. It's out of scope of this PR to fix this, but worth noting that adding `* item` won't actually add items to a proper list.